letters/letters.py

```diff
@@ -12,41 +15,64 @@ class Letters(commands.Cog):
     @commands.command()
     async def letters(self, ctx, *, msg):
         """Outputs large emote letters (\"regional indicators\") from input text.
-        Accepts a-z, 0-9, and whitespace only.
+
+        The result can be outputted as raw emote code using `-raw` flag.
 
         Example:
         - `[p]letters I would like this text as emotes 123`
+        - `[p]letters -raw I would like this text as raw emote code 123`
         """
 
         # Grab message content
         input = msg.lower()
 
-        # Ensure it doesn't contain any special chars, numbers, etc.
-        regexp = re.compile(r'[^a-z0-9 ]')
+        # Check for raw flag
+        if input.startswith("-raw"):
+            raw = True
+            input = input.lstrip("-raw").lstrip()
+
+        else:
```

Comment:
This else could be simplified to instantiating `raw` as False above the if statement